client/post-editor/editor-featured-image/index.jsx

```diff
@@ -125,4 +138,15 @@ export default React.createClass( {
 			</Button>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, ownProps ) => {
+		const { post, site } = ownProps;
```

Comment:
@gwwar here's what I have currently:

```js
const { post, site } = ownProps;
const siteId = site && site.ID;
const featuredImageId = getFeaturedImageId( post );

return {
	featuredImage: getMediaItem( state, siteId, featuredImageId ),
};
```

Is that what you meant since `getMediaItem` only requires `siteId`? I wasn't sure why we needed to also return `siteId` in your comment.